cmd/metrics-server/app/options/options.go

```diff
@@ -168,8 +168,8 @@ func (o Options) ApiserverConfig() (*genericapiserver.Config, error) {
 	serverConfig.OpenAPIV3Config.Info.Title = "Kubernetes metrics-server"
 	serverConfig.OpenAPIConfig.Info.Version = strings.Split(versionGet.String(), "-")[0] // TODO(directxman12): remove this once autosetting this doesn't require security definitions
 	serverConfig.OpenAPIV3Config.Info.Version = strings.Split(versionGet.String(), "-")[0]
-	serverConfig.EffectiveVersion = utilversion.DefaultKubeEffectiveVersion()
 
+	serverConfig.EffectiveVersion = utilcompatibility.DefaultBuildEffectiveVersion()
```

Comment:
Due to https://github.com/kubernetes/component-base/commit/77439d2cc6f9c66f8cb35f62b25722ce7b0ea6a1#diff-f6bf0ae0d21c9408c6624174da11dafb369ce306f2cdcea704285d8acc87e19e